tests/analyze/test_plot_correlation.py

```
@@ -0,0 +1,187 @@
+    "triangle",
+    [
+        "lower",
+        "upper"
+    ]
+)
+def test_diagonal_heatmap(correlation_matrix, triangle):
+    """Test making a heatmap from a correlation matrix with diagonal set to True"""
+    corr_fig = plotCorrelationHeatmap(correlation_matrix, diagonal=True, triangle=triangle)
+    assert isinstance(corr_fig, Figure), \
+        "Wrong return type, expect a matplotlib Figure"
+    assert corr_fig.get_suptitle() == f"Correlation Heatmap", \
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)